go.mod

```diff
@@ -40,7 +40,7 @@ require (
 	k8s.io/api v0.29.2
 	k8s.io/apimachinery v0.29.2
 	k8s.io/client-go v1.5.2
-	k8s.io/utils v0.0.0-20240102154912-e7106e64919e
+	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
```

Comment:
when we bump k8s again this get overrides 
so do we need to handle this again ??

Reply:
removed those changes